### ISSUE TITLE
How can the sum of cross entropy loss and Dice loss during training be negative?

### ISSUE THREAD

**Issue by SvyJ** [duweidai/TransHRNet#4]
Thanks for your amazing work. However, I encountered confusion when I read the related article entitled "3D Medical image segmentation using parallel transformers", that is, the network was trained using binary Cross-Entropy loss and Dice loss, but Fig. 10 shows that the sum of losses has fallen below 0, can you clarify this phenomenon?
